test/integration/targets/reboot/tasks/main.yml

```diff
@@ -75,6 +78,27 @@
       vars:
         timeout: "{{ timeout_value[ansible_facts['distribution'] | lower] | default(60) }}"
 
+    - name: Test molly-guard
+      block:
+        - import_tasks: get_boot_time.yml
+
+        - name: Install molly-guard
+          apt:
+            update_cache: yes
+            name: molly-guard
+            state: present
+
+        - name: Reboot when molly-guard is installed
+          reboot:
+            extra_search_paths: /lib/molly-guard
+          register: reboot_result
+
+        - import_tasks: check_reboot.yml
+
+      when: ansible_facts.pkg_mgr == 'apt'
```

Comment:
maybe ansible_distribution in ['debian','ubuntu'] - we don't know if other distros might not have that package...